lib/virt_autotest/utils.pm

```diff
@@ -64,6 +64,23 @@ sub is_pv_guest {
     return $guest =~ /\bpv\b/;
 }
 
+#Check if guest is SLE with optional filter for:
+#Version: <=12-sp3 =12-sp1 >11-sp1 >=15 15+ (>=15 and 15+ are equivalent)
+#usage: guest_is_sle($guest_name, '<=12-sp2')
+sub guest_is_sle {
+    my $guest_name = lc shift;
+    my $query = shift;
+
+    return 0 unless $guest_name =~ /sle/;
+    return 1 unless $query;
```

Comment:
Do returned 0 and 1 represent errors ? If yes, consider choose other values, I do not think they are suitable, especially 0.

Reply:
I copied the idea from 'use version_utils qw(is_sle)`. the function 'is_sle' is always used this way:
`if (is_opensuse || (is_sle && is_sut_reg)) {`
`*** if is_sle('<=12-sp4')`
rc=1/0 works this way.